addons/point_of_sale/tools/posbox/configuration/upgrade.sh

```diff
@@ -79,13 +93,26 @@ download_raspios () {
             echo "Error_Raspios_Download"
             exit 0
         fi
-        unzip "${RASPIOS}"
+        pkill chromium # kill chromium to free space
+        rm -rf /tmp/0  # clean chrome workspace to free space
+        printf "Uncompressing %s..." "${RASPIOS}"
+        touch .uncompressing
```

Comment:
`.uncompressing` is used as a flag for the second method (2nd ajax call) to wait until end of decompression if timeout stops current ajax request